### PR TITLE
Fix open issues

### LINKED ISSUE #25182: Versioninh explanation is inconsistent with the versioning table
The table and the explanation are a bit confusing.
Looking at the table I had the idea that SDK minor (which is multiplied by 100) is related to the runtime patch level.
That is not the case.

I actually was expecting the initial version of the SDK to be 5.0.0 instead of 5.0.100.
I got most confused by the last line in which and SDK feature change also bumped the runtime version.
I would expect that a runtime version of 5.0.0 with and SDK version of 5.0.200 is also allowed.
I think that would be clearer.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4723eb4d-c4b9-f44c-7753-caa4a3c8cfdd
* Version Independent ID: 17270776-9bf5-fe60-f9a7-43311e5b4654
* Content: [How the .NET Runtime and SDK are versioned - .NET](https://docs.microsoft.com/en-us/dotnet/core/versions/)
* Content Source: [docs/core/versions/index.md](https://github.com/dotnet/docs/blob/main/docs/core/versions/index.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### LINKED ISSUE #34273: Duration of support
The article claims:

> .NET releases a new major version every November. Even-numbered releases, such as .NET 6 or .NET 8, are long-term supported (LTS). 

OK so far.

> Odd-numbered releases are supported until the next major release.

Not quite right. According to <https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core#cadence>, odd-numbered (STS) major releases are supported for 18 months, i.e. 6 months past the next (LTS) major release.

> The latest release of .NET is .NET 6.

No, .NET 7 was already released.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 4723eb4d-c4b9-f44c-7753-caa4a3c8cfdd
* Version Independent ID: 17270776-9bf5-fe60-f9a7-43311e5b4654
* Content: [How the .NET Runtime and SDK are versioned - .NET](https://learn.microsoft.com/en-us/dotnet/core/versions/)
* Content Source: [docs/core/versions/index.md](https://github.com/dotnet/docs/blob/main/docs/core/versions/index.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

---
[Associated WorkItem - 66232](https://dev.azure.com/msft-skilling/Content/_workitems/edit/66232)

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #25182
Fixes #34273

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/core/versions/index.md](https://github.com/dotnet/docs/blob/e946659b868244c5f8547789d60cafe41ac6ec4e/docs/core/versions/index.md) | [Overview of how .NET is versioned](https://review.learn.microsoft.com/en-us/dotnet/core/versions/index?branch=pr-en-us-35170) |

### CODE REVIEW

File: docs/core/versions/index.md [hunk 1]
```diff
@@ -20,12 +20,19 @@ The first two positions of the .NET SDK version number match the .NET Runtime ve
...
+|-----------------------|-------------------|-------------------|-------------------------------------------------------------------------------------|
+| Initial release       | 5.0.0             | 5.0.100           | Initial release.                                                                    |
+| SDK patch             | 5.0.0             | 5.0.101           | Runtime didn't change with this SDK patch. SDK patch bumps last digit in SDK patch. |
+| Runtime and SDK patch | 5.0.1             | 5.0.102           | Runtime patch bumps Runtime patch number. SDK patch bumps last digit in SDK patch.  |
+| SDK feature change    | 5.0.1             | 5.0.200           | Runtime patch didn't change. New SDK feature bumps first digit in SDK patch.        |
+| Runtime patch         | 5.0.2             | 5.0.102           | Runtime patch bumps Runtime patch number. SDK doesn't change.                       |
+
+From the preceding table you can see several policies:
+
+- The Runtime and SDK share major and minor versions. The first two numbers for a given SDK and runtime should match. All the preceding examples are part of the .NET T5.0 release stream.
```
Comment (tdykstra) [inline docs/core/versions/index.md:33]:
```suggestion
- The Runtime and SDK share major and minor versions. The first two numbers for a given SDK and runtime should match. All the preceding examples are part of the .NET 5.0 release stream.
```

**Review by tdykstra — APPROVED** [review]
The examples and explanations make it much clearer. I noticed one typo.

**Review by tdykstra — COMMENTED** [review]
One follow-up thought -- I wonder if machine translation will know what to do with the word "rev"? It seems like jargon but it might be fine.

File: docs/core/versions/index.md [hunk 2]
```diff
@@ -20,12 +20,19 @@ The first two positions of the .NET SDK version number match the .NET Runtime ve
...
+| SDK patch             | 5.0.0             | 5.0.101           | Runtime didn't change with this SDK patch. SDK patch bumps last digit in SDK patch. |
+| Runtime and SDK patch | 5.0.1             | 5.0.102           | Runtime patch bumps Runtime patch number. SDK patch bumps last digit in SDK patch.  |
+| SDK feature change    | 5.0.1             | 5.0.200           | Runtime patch didn't change. New SDK feature bumps first digit in SDK patch.        |
+| Runtime patch         | 5.0.2             | 5.0.102           | Runtime patch bumps Runtime patch number. SDK doesn't change.                       |
+
+From the preceding table you can see several policies:
+
+- The Runtime and SDK share major and minor versions. The first two numbers for a given SDK and runtime should match. All the preceding examples are part of the .NET 5.0 release stream.
+- The patch version of the runtime revs only when the runtime is updated. The SDK patch number doesn't update for a runtime patch.
+- The patch version of the SDK revs only when the SDK is updated. It's possible that a runtime patch doesn't require an SDK patch.
```
Comment (BillWagner) [inline docs/core/versions/index.md:35]: remove jargon:

```suggestion
- The patch version of the SDK updates only when the SDK is updated. It's possible that a runtime patch doesn't require an SDK patch.
```

**Comment by BillWagner** [comment]
closing and reopening for a fresh build

**Pull request merged**